src/colab_superarchives/views.py

```diff
@@ -452,6 +461,15 @@ def get_context_data(self, **kwargs):
                     checked
                 ))
 
+            paginator = Paginator(lists, per_page)
```

Comment:
@matheus-fga these lines related to pagination were moved to https://github.com/colab/colab-superarchives-plugin/blob/master/src/colab_superarchives/views.py#L501 after some code refactoring